hoomd/mpcd/pytest/test_update.py

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2009-2024 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import pytest
+import hoomd
+import numpy as np
+
+from hoomd.conftest import pickling_check
+from hoomd.logging import LoggerCategories
+from hoomd.conftest import logging_check
+
+
+@pytest.fixture
+def snap():
+    snap = hoomd.Snapshot()
+    if snap.communicator.rank == 0:
+        snap.configuration.box = [20, 20, 20, 0, 0, 0]
+        snap.particles.N = 0
+        snap.particles.types = ["A"]
+        snap.mpcd.types = ["A"]
+        snap.mpcd.mass = 1.0
+    return snap
+
+
+@pytest.fixture
+def flow():
+    flow = hoomd.mpcd.update.ReverseNonequilibriumShearFlow(
+        trigger=hoomd.trigger.Periodic(1),
```

Comment:
Will this work with just the number 1?
```suggestion
        trigger=1,
```